### ISSUE TITLE
Create a syslogd server and store data in history/influx/sql

### ISSUE THREAD

**Issue by stabilostick** [ioBroker/AdapterRequests#93]
Usfull for analysis, monitoring and alerting of various devices and systems. Might be acting as man-in-the-middle (gateway) for alerting/filtering with ioBroker and still using an enterprise level monitoring system backend as an option. Must support RFC 5424.

**Comment by Apollon77** [comment]
What exactly is the idea?! You can log into syslog already. 

Or is the idea to process syslog data via iobroker?

**Comment by stabilostick** [comment]
Yes. This adapter enables ioBroker to act like a syslog server. And you can use the script adapter as a rule engine.

**Comment by Apollon77** [comment]
Maybe also look at "Parser" or "systeminfo" adapter ... both already can do log parsing and stuff ...

**Comment by aurodionov** [comment]
Тут скорее нужны настройки syslog в адаптере admin

**Comment by mcm1957** [comment]
[Stabilostick](https://github.com/Stabilostick)
Is this adapter still work in progress, already released or has development been cancelled?
No update on status sinc 2018
No specific adaptername / git repo link here

**Comment by mcm1957** [comment]
No (matching) repository visible at [Stabilostick](https://github.com/Stabilostick)
No feedback to question dated Nov 20,2022

Looks like this adapter does no longer exist or ist in-progress.
@Stabilostick please respong if I'm incorrect.

resetting state
